channels/us.m3u

```diff
@@ -1184,7 +1184,7 @@ https://dai.google.com/linear/hls/event/KsvJAc81Qoewj6opYso6Fw/master.m3u8
 https://stitcheraws.unreel.me/wse-node02.powr.com/live/5c7e25a1932c8368bdbfd87d/playlist.m3u8
 #EXTINF:-1 tvg-id="EntertainmentTonight.us" tvg-name="Entertainment Tonight" tvg-country="US" tvg-language="English" tvg-logo="https://pmcdeadline2.files.wordpress.com/2018/10/et-live1-e1541006731547.png" group-title="News",Entertainment Tonight (1080p)
 https://etlive-mediapackage-fastly.cbsaavideo.com/dvr/manifest.m3u8
-#EXTINF:-1 tvg-id="Entrepeneur.us" tvg-name="Entrepeneur" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Entrepeneur
+#EXTINF:-1 tvg-id="Entrepeneur.us" tvg-name="Entrepeneur" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/Wb4WPPK.jpg" group-title="Series",Entrepeneur
```

Comment:
Is there a typo on the name ?

Should it be named "Entrepreneur" ?

Reply:
You are correct. I will fix that.